worlds/oot_soh/LogicHelpers.py

```diff
@@ -995,7 +999,7 @@ def can_build_rainbow_bridge(bundle: tuple[CollectionState, Regions, "SohWorld"]
     if has_item(Items.GREG_THE_GREEN_RUPEE, bundle) and world.options.rainbow_bridge_greg_modifier.value == 1:
         greg_reward = 1
 
-    bridge_setting = world.options.rainbow_bridge.value # TODO: Remove magic number in comparisons, replace with enum
+    bridge_setting = world.options.rainbow_bridge.value
 
     return (bridge_setting == 1 or
             (bridge_setting == 0 and has_item(Items.SHADOW_MEDALLION, bundle) and has_item(Items.SPIRIT_MEDALLION, bundle) and can_use(Items.LIGHT_ARROW, bundle)) or
```

Comment:
```suggestion
    return (world.options.rainbow_bridge or
            (has_item(Items.SHADOW_MEDALLION, bundle) and has_item(Items.SPIRIT_MEDALLION, bundle) and can_use(Items.LIGHT_ARROW, bundle)) or
```
The intent here was probably to just load world.options.rainbow_bridge directly into the return

Reply:
of course, this seems a little out of scope for this PR, but it would be appreciated if it was included anyway